lib/jumubase_web/templates/performance/new.html.eex

```diff
@@ -20,7 +20,7 @@
             class="form-control"
             v-model="contest_category_id"
             v-on:change="contestCategoryChanged">
-            <option disabled value=""><%= gettext("Please choose") %></option>
```

Comment:
@denisw I'd like to keep this because it prevents users from clearing their choice after selecting a CC, but I noticed a problem: When submitting a completely empty form (no CC selected, zero appearances, zero pieces), the `"performances"` key is missing completely from the request params (despite the CC field still being present in the form). Any idea why this happens?

Reply:
@richeterre Is `performance` missing already from the form request (according to browser devtools) or only from the parsed result on the server side? Perhaps Phoenix just interpreted all the unset values as "there is no data at all".

If this is a problem, how about making the empty option enabled initially, but disable it once the user has selected anything else?

Reply:
@denisw Yes, it's missing already from the request and apparently that's a feature of the `disabled` attribute: https://stackoverflow.com/questions/1191113/how-to-ensure-a-select-form-field-is-submitted-when-it-is-disabled (in that question the whole `<select>` is disabled, but apparently disabling the selected `<option>` has the same effect).

A simple workaround is to add a hidden field to the form:

`<input type="hidden" name="performance" value="" />`